deploy/examples/create-external-cluster-resources.py

```diff
@@ -1758,6 +1793,35 @@ def gen_json_out(self):
                     },
                 }
             )
+        else:
+            if self.out_map["RBD_METADATA_EC_POOL_NAME"]:
+                json_out.append(
+                    {
+                        "name": "ceph-rbd",
+                        "kind": "StorageClass",
+                        "data": {
+                            "dataPool": self.out_map["RBD_POOL_NAME"],
+                            "pool": self.out_map["RBD_METADATA_EC_POOL_NAME"],
+                            "csi.storage.k8s.io/provisioner-secret-name": f"rook-{self.out_map['CSI_RBD_PROVISIONER_SECRET_NAME']}",
+                            "csi.storage.k8s.io/controller-expand-secret-name": f"rook-{self.out_map['CSI_RBD_PROVISIONER_SECRET_NAME']}",
+                            "csi.storage.k8s.io/node-stage-secret-name": f"rook-{self.out_map['CSI_RBD_NODE_SECRET_NAME']}",
+                        },
+                    }
+                )
+            else:
+                json_out.append(
+                    {
+                        "name": "ceph-rbd",
+                        "kind": "StorageClass",
+                        "data": {
+                            "pool": self.out_map["RBD_POOL_NAME"],
```

Comment:
If a rados namespace is specified, the pool name would need to be generated differently as mentioned in [this comment](https://github.com/rook/rook/pull/13196#discussion_r1394585714). Are you planning on generating that?

Reply:
That is taken care in ocs operator for downstream and upstream in the shell script

Reply:
Oh this is for the storage class, I was confusing it with the rados namespace above